app/main.py

```diff
@@ -1,6 +1,17 @@
 from typing import Callable
+from typing import Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    args_dict = {}
+
+    def saver(*args) -> Any:
```

Comment:
You could use decorator from `functools` for `saver` to preserve the wrapped function name and docstring.